fluentd/datadog_checks/fluentd/data/conf.yaml.example

```
@@ -1,4 +1,20 @@
 init_config:
+    ## @param skip_proxy - boolean - optional - default: false
+    ## Set `skip_proxy` parameter to `true` bypass any proxy settings
```

```suggestion
    ## Set `skip_proxy` parameter to `true` bypass any proxy
```
`settings` used twice remove from this line or the line below